mesh/nurbs.cpp

```diff
@@ -1122,6 +1122,62 @@ void NURBSPatch::SwapDirections(int dir1, int dir2)
    swap(newpatch);
 }
 
+
```

Comment:
Extra line

Reply:
I have removed the extra line in [ede8ab9](https://github.com/mfem/mfem/pull/3226/commits/ede8ab9a0b61c7d3a8f54e5095b86a832525d87b)